Tests/FluentAssertions.Specs/Primitives/DateTimeAssertionSpecs.cs

```diff
@@ -139,6 +139,20 @@ public void Should_succeed_when_asserting_datetime_value_is_equal_to_the_same_va
             act.Should().NotThrow();
         }
 
+        [Fact]
+        public void Should_succeed_when_asserting_datetime_value_is_equal_to_the_same_nullable_value()
+        {
+            // Arrange
+            DateTime dateTime = new DateTime(2016, 06, 04);
```

Comment:
🔧 Would be nice to rewrite this as `4.June(2016)`. Same for the other places.

Reply:
Done!